blue-web/src/main/resources/webapp/js/paper/directives/TocDirective.js

```diff
@@ -62,12 +64,23 @@ angular.module('bluelatex.Paper.Directives.Toc', [])
           }
 
           currentlevel = line.level;
+          if(!numbers[currentlevel]) {
+            numbers[currentlevel] = 1;
+          } else {
+            numbers[currentlevel] ++;
+          }
+          var lineNumber = "";
+          for (var j = 1; j <= currentlevel; j++) {
+            if(numbers[j]) {
+              lineNumber += numbers[j] + ".";
+            }
+          }
```

Comment:
Wouldn't it be appropriate to use [`join`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) here?